R/geom-point.R

```diff
@@ -188,6 +186,9 @@ GeomPoint <- ggproto("GeomPoint", Geom,
 #' # Strings with 1 or less characters are interpreted as symbols
 #' translate_shape_string(c("a", "b", "?"))
 translate_shape_string <- function(shape_string) {
```

Comment:
Shouldn't this be `validate_shape_string()` per the new focused vocabulary?

Reply:
Good point, yes I think it should in principle. However, it is exported so it isn't really an internal function.

Reply:
ah - let's keep it asis